.github/workflows/breaking-change-check.yml

```diff
@@ -0,0 +1,96 @@
+name: Breaking Change Check
```

Comment:
Is it necessary to be its own action? (Does it take a long time to perform the check?) Or could we just have this in the CI.yml where we have the treeshaking and typescript checks?

Reply:
Oh because it builds base as well as current then compares - sounds good!

Reply:
I guess we might need more machines if that is possible... cause now we have 2 teams conributing to the repo, imagine if we have more teams but we still don't have enough machines to run all PRs